command/root.go

```diff
@@ -95,6 +93,18 @@ var RootCmd = &cobra.Command{
 
 	SilenceErrors: true,
 	SilenceUsage:  true,
+
+	Annotations: map[string]string{
+		"help:examples": `
```

Comment:
Is it not possible to use the `Example` field on `Command` for examples?

I'm :+1: on extending the help like this to add sections `cobra` doesn't yet support, but it seems like when possible we should use existing struct fields.

Reply:
🤦 yes... I should be using the Example field